docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+services:
+  db:
+    image: postgres:14.15-alpine3.21
+    restart: always
+    env_file:
+      - .env
+    ports:
+      - "5432:5432"
+    volumes:
+      - my_db:$PGDATA
+
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py wait_for_db &&
+      python manage.py migrate &&
+      python manage.py runserver 0.0.0.0:8000"
```

Comment:
The command sequence ensures that the database is ready before running migrations and starting the server. This is a good practice to avoid race conditions when starting the application.